app/assets/javascripts/discourse/app/components/modal/group-add-members.hbs

```diff
@@ -0,0 +1,61 @@
+<DModal
+  @title={{this.title}}
+  @closeModal={{@closeModal}}
+  class="group-add-members-modal"
+  @flash={{this.flash}}
+  @flashType="danger"
```

Comment:
Does `danger` work correctly? I think we want `error` here? (which maps to the (`.alert-error`](https://github.com/discourse/discourse/blob/9ee1972a2cbc92e72f3dc36d0624eb4b07ed50a7/app/assets/stylesheets/common/base/alert.scss#L18-L37) class)

Reply:
Removed the explicit class definition here, and addressed the larger problem in https://github.com/discourse/discourse/pull/22416